ios/RNSensorsAccelerometer.m

```diff
@@ -49,12 +49,12 @@ - (void) isAvailableWithResolver:(RCTPromiseResolveBlock) resolve
         {
             resolve(@YES);
         } else {
-            reject(@"-1", @"Accelerometer is not active", nil);
+            resolve(@NO);
```

Comment:
Is this chance necessary to stop the crashes? A reject can be handled by a .catch().